pkgs/by-name/ul/ultraleap-hand-tracking-control-panel/package.nix

```
@@ -0,0 +1,96 @@
+{
+  addDriverRunpath,
+  autoPatchelfHook,
+  dbus,
+  dpkg,
+  fetchurl,
+  lib,
+  libGL,
+  libxkbcommon,
+  makeWrapper,
+  stdenv,
+  udev,
+  ultraleap-hand-tracking-service,
+  vulkan-loader,
+  wayland,
+  xorg,
+  zlib,
+}:
+stdenv.mkDerivation {
+  pname = "ultraleap-hand-tracking-control-panel";
+  version = "3.4.1";
+
+  src = fetchurl {
+    url = "https://repo.ultraleap.com/apt/pool/main/u/ultraleap-hand-tracking-control-panel/ultraleap-hand-tracking-control-panel_1125862.deb";
+    hash = "sha256-Pli0ENEtzBQGvfo2nIf09b62Xt3gzML/QYXMwtrD+7Y=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    makeWrapper
```

```suggestion
    makeBinaryWrapper
```
We don't need the overhead of a script based wrapper here